model_validation/ciff_sam_results.py

```diff
@@ -86,15 +86,21 @@ def compute_mam_duration(self, strata=DEFAULT_STRATA):
 
 project_results_directory = '/ihme/costeffectiveness/results/vivarium_ciff_sam'
 
+# Eventually, I want to write a script to recursively search the directories and
+# populate this table automatically instead of hardcoding it. E.g. use Path.rglob to
+# find all output.hdf files and/or count_data subdirectories in a specified directory:
+# https://stackoverflow.com/questions/2186525/how-to-use-glob-to-find-files-recursively
+# https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob
 models = pd.DataFrame(
     [
-        [2.3, 'v2.3_wasting_birth_prevalence', '2021_07_26_17_14_31'],
-        [2.4, 'v2.4_corrected_fertility', '2021_08_03_15_08_32'],
-        [2.5, 'v2.5_stunting', '2021_08_05_16_17_12'],
-        [3.0, 'v3.0_sq_lns', '2021_08_16_17_54_19'],
-        [3.1, 'v3.1_sq_lns_stunting_stratified', '2021_08_24_10_28_32'],
-        [4.0, 'v4.0_wasting_treatment', '2021_09_20_14_45_25'],
-        [4.1, 'v4.1_wasting_treatment', '2021_09_24_16_36_30'],
+        ['2.3', 'v2.3_wasting_birth_prevalence', '2021_07_26_17_14_31'],
+        ['2.4', 'v2.4_corrected_fertility', '2021_08_03_15_08_32'],
+        ['2.5', 'v2.5_stunting', '2021_08_05_16_17_12'],
+        ['3.0', 'v3.0_sq_lns', '2021_08_16_17_54_19'],
+        ['3.1', 'v3.1_sq_lns_stunting_stratified', '2021_08_24_10_28_32'],
+        ['4.0', 'v4.0_wasting_treatment', '2021_09_20_14_45_25'],
+        ['4.1', 'v4.1_wasting_treatment', '2021_09_24_16_36_30'],
+        ['4.5.2', 'v4.5.2_x_factor', '2021_09_29_12_12_47'],
```

Comment:
Hmm, I guess that could be useful in that it would reduce the amount of typing necessary to get to the right directory, but we usually just copy/paste the directory from Slack anyway, and I think the names are already fairly descriptive. What I'm really going for here is a list of all the model directories that I can access from within Python to easily load any model version I want without having to look them up separately in Slack or from other notebooks. I guess if we had symlinks to the right subfolders in the top model results directory, that would make it easier to go directly to the model we want by doing `!ls` from within a notebook.